### PR TITLE
Implement Cardano transactions verification in `mithril-client-wasm`

### LINKED ISSUE #1470: Implement `Cardano Transactions` verification in the WASM client
## Why
The WASM Mithril client should implement a function to certify a transaction list.

## What
Implement the new function to certify transactions in the WASM Mithril client API.

## How
- [x] Adapt the `mithril-aggregator-fake` to handle the new types of data
- [x] Add a function for generation/verification of a proof for a list of transactions
- [x] Add tests that call this functions using the `mithril-aggregator-fake`
- [x] Update the `www` example
- [x] Update documentation

### PR DESCRIPTION
**Pull request by dlachaumepalo**
## Content
This PR includes an update of `mithril-client-wasm` to handle the Cardano transactions functionnalities exposed by `mithril-client` crate.

In order to implement Cardano transaction tests in the WASM library, the mechanism that sets up the default data of `mithril-aggregator-fake` had to be improved. Build scripts now generate useful Rust code for testing, containing all default data. Data can be generated by the `import.sh` script of the `mithril-aggregator-fake` and stored in JSON files located in `mithril-test-lab/mithril-aggregator-fake/default_data`.

## Pre-submit checklist

- Branch
  - [ ] Tests are provided (if possible)
  - [x] Crates versions are updated (if relevant)
  - [x] Commit sequence broadly makes sense
  - [x] Key commits have useful messages
- PR
  - [x] No clippy warnings in the CI
  - [x] Self-reviewed the diff
  - [x] Useful pull request description
  - [x] Reviewer requested
- Documentation
  - [x] Update README file (if relevant)
  - [x] Update documentation website (if relevant)
  - [ ] Add dev blog post (if relevant)

## Issue(s)
Closes #1470

### CODE REVIEW

**Review by Alenar — APPROVED** [review]

File: mithril-build-script/Cargo.toml
[the remark this answers is not in the document]
Comment (dlachaumepalo) [reply mithril-build-script/Cargo.toml:1]: After our discussion, we'll add a README file and a description to the crate `mithril-build-script`.

File: mithril-build-script/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 [package]
 name = "mithril-build-script"
 version = "0.1.0"
+description = "A toolbox of generations functions for Mithril crates build scripts"
```
Comment (jpraynaud) [inline mithril-build-script/Cargo.toml:4]:
```suggestion
description = "A toolbox for Mithril crates build scripts"
```

File: mithril-build-script/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Mithril-build-script
+
+A toolbox of generations functions for Mithril crates that needs a [build scripts phase](https://doc.rust-lang.org/cargo/reference/build-scripts.html).
+
+In addition this crates allow to tests those functions as tests are not supported directly in `build.rs` (see: [rust-lang/cargo#1581](https://github.com/rust-lang/cargo/issues/1581)).
```
Comment (jpraynaud) [inline mithril-build-script/README.md:5]:
```suggestion
A toolbox for Mithril crates that need a [build scripts phase](https://doc.rust-lang.org/cargo/reference/build-scripts.html).

Moreover, this crate allows to test the build scripts, which is not supported directly in `build.rs` (see: [rust-lang/cargo#1581](https://github.com/rust-lang/cargo/issues/1581)).
```

**Review by jpraynaud — APPROVED** [review]
LGTM :+1: 

I commented few typos to fix before merging.

**Pull request merged**